ui/snippets/searchBar/SearchBarSuggest/SearchBarSuggestItem.tsx

```diff
@@ -112,12 +119,19 @@ const SearchBarSuggestItem = ({ data, isMobile, searchTerm, onClick, addressForm
       case 'ens_domain': {
         return <SearchBarSuggestDomain data={ data } searchTerm={ searchTerm } isMobile={ isMobile } addressFormat={ addressFormat }/>;
       }
+      case 'cluster': {
+        return <SearchBarSuggestCluster data={ data } searchTerm={ searchTerm } isMobile={ isMobile } addressFormat={ addressFormat }/>;
+      }
       case 'tac_operation': {
         return <SearchBarSuggestTacOperation data={ data } searchTerm={ searchTerm } isMobile={ isMobile } addressFormat={ addressFormat }/>;
       }
     }
   })();
 
+  if (data.type === 'cluster' && !isClusterClickable) {
+    return content;
+  }
```

Comment:
Let's move the `isClusterClickable` condition closer to this statement and rename it to `hasLink` to make it more generic, as we might alter it for other purposes later.

```suggestion
  if (!hasLink) {
    return content;
  }
```